lib/matplotlib/ticker.py

```diff
@@ -798,16 +798,67 @@ def _formatSciNotation(self, s):
 
 class LogFormatter(Formatter):
     """
-    Format values for log axis.
+    Base class for formatting ticks on a log or symlog scale.
+
+    It may be instantiated directly, or subclassed.
+
+    Parameters
+    ----------
+    base : float, optional, default: 10.
+        Base of the logarithm used in all calculations.
+
+    labelOnlyBase : bool, optional, default: False
+        If True, label ticks only at integer powers of base.
+        This is normally True for major ticks and False for
+        minor ticks.
+
+    minor_thresholds : (subset, all), optional, default: (1, 0.4)
```

Comment:
It could go either way; I chose this because it corresponds to the order in which the thresholds are checked: first see if any minors are labeled, and if so, see if all of them are. The most common case is expected to be that none are labeled.